eng/CHECKENFORCER

```diff
@@ -1,2 +1,2 @@
 format: v0.1-alpha
-minimumCheckRuns: 1
```

Comment:
This looks like a bad merge, please undo. fyi @mitchdenny

Reply:
Done.